src/main/webapp/app/modules/canvas/filter/filter-element.tsx

```diff
@@ -206,15 +199,12 @@ const mapStateToProps = (storeState: IRootState) => ({
   view: storeState.views.entity,
   filterData: storeState.visualmetadata.filterData,
   featuresList: storeState.feature.entities,
-  selectedFilter: storeState.filter.selectedFilters,
-  filterStateChange: storeState.filter.filterStateChange,
-  filters: storeState.filter.paramObject,
-
+  selectedFilters: storeState.filter.selectedFilters,
+  isUpdateValueInFilter: storeState.filter.isUpdateValueInFilter,
```

Comment:
isUpdateValueInFilter...what is the use of this state? if it is required then check the suitable name....